common/app/views/fragments/inlineJSBlocking.scala.html

```diff
@@ -63,30 +58,11 @@
 
     // ************* LOAD THE MAIN APP ASYNC *************
 
-    @if(conf.Configuration.assets.useHashedBundles) {
```

Comment:
Do we no longer need [useHashedBundles](https://github.com/guardian/frontend/blob/531b718e2950d9ac9abd5f2dcb7221ce8576a69f/common/app/common/configuration.scala#L323-L323) then?